tests/server_test.go

```diff
@@ -9857,6 +9858,111 @@ func TestServer_Prometheus_Write(t *testing.T) {
 	}
 }
 
+func TestCliEndToEnd(t *testing.T) {
+	config := NewConfig()
+	config.HTTPD.FluxEnabled = true
+	s := OpenServer(config)
+	defer s.Close()
+
+	// Some test data
+	s.CreateDatabase(t.Name())
+	defer s.DropDatabase(t.Name())
+	points, err := models.ParsePointsString(`
+	m0,k=k0 f=0i 0
+	m0,k=k0 f=1i 1
+	m0,k=k0 f=2i 2
+	m0,k=k0 f=3i 3
+	m0,k=k0 f=4i 4
+	m0,k=k1 f=5i 5
+	m0,k=k1 f=6i 6
+	m1,k=k0 f=5i 7
+	m1,k=k2 f=0i 8
+	m1,k=k0 f=6i 9
+	m1,k=k1 f=6i 10
+	m1,k=k0 f=7i 11
+	m1,k=k0 f=5i 12
+	m1,k=k1 f=8i 13
+	m1,k=k2 f=9i 14
+	m1,k=k3 f=5i 15`)
+	assert.NoError(t, err)
+	assert.NoError(t, s.WritePoints(t.Name(), "autogen", models.ConsistencyLevelAny, nil, points))
```

Comment:
Any chance that `ConsistencyLevelAny` will create flakiness in the test?

Reply:
We don't honour the consistency level in 1.x OSS, only in enterprise - https://archive.docs.influxdata.com/influxdb/v1.2/tools/api/#query-string-parameters-1